lua/astrocommunity/pack/full-dadbod/init.lua

```diff
@@ -20,12 +29,17 @@ return {
                   {
                     event = "FileType",
                     desc = "dadbod completion",
-                    pattern = { "sql", "mysql", "plsql" },
+                    pattern = { "sql", "mysql", "plsql", "pg" },
```

Comment:
Hmmm I asked ChatGPT and it is a file type for Postgres. Commonly used with pgtap

Reply:
Can keep it removed. I can add it myself

Reply:
You were setting the filetype in your autocommand of `pg` to `sql` so that pattern would never have been satisfied in the editor. You can add it yourself, but as you configured it the filetype will be `sql` which will connect it to sql tooling and treesitter and this pattern will never be met.

Also looking at something like `pgsql.vim` which is the vim plugin for postgresql support, they also set the filetype to `sql`. So I think you should probably re-evaluate if ChatGPT knows what it's saying 🤡